tests/lib/rules/no-fallthrough.js

```diff
@@ -63,7 +63,7 @@ ruleTester.run("no-fallthrough", rule, {
         "switch (foo) { case 0: try {} finally { break; } default: b(); }",
         "switch (foo) { case 0: try { throw 0; } catch (err) { break; } default: b(); }",
         "switch (foo) { case 0: do { throw 0; } while(a); default: b(); }",
-        "switch (foo) { case 0: a(); \n// eslint-disable-next-line no-fallthrough\n case 1: }",
+        "switch (foo) { case 0: a(); \n// eslint-disable-next-line rule-to-test/no-fallthrough\n case 1: }",
```

Comment:
This test was added in https://github.com/eslint/eslint/commit/b4f8329164d7b293a1557e05b987d2a685fe1d30 to address https://github.com/eslint/eslint/issues/16650, not sure why it isn't working with FlatRuleTester.

Reply:
Ah, got it. This is the id of the rule being tested.

https://github.com/eslint/eslint/blob/90077199fe519953f9af8664bf947db4e4958514/lib/rule-tester/flat-rule-tester.js#L519